src/utils/airseeker-validation.ts

```diff
@@ -1,5 +1,5 @@
-// NOTE Copied from Airseeker
 import { z } from 'zod';
```

Comment:
Airseeker validator is going to have a few refinements added. Not sure this might have an impact here tho. https://github.com/api3dao/airseeker/pull/76

Reply:
will port those over once its finalized

Reply:
We should be able to import the validation from Airseeker's published package. The only reason we copied it was because we wanted to avoid building it here.